python/plugins/db_manager/db_manager.py

```diff
@@ -448,8 +448,8 @@ def setupUi(self):
         sep.setVisible(False)
 
         self.actionImport = self.menuTable.addAction(QIcon(":/db_manager/actions/import"),
-                                                     self.tr("&Import layer/file"), self.importActionSlot)
-        self.actionExport = self.menuTable.addAction(QIcon(":/db_manager/actions/export"), self.tr("&Export to file"),
+                                                     self.tr("&Import Layer/file…"), self.importActionSlot)
```

Comment:
Should it not be *File*, given https://github.com/qgis/QGIS/pull/6325/files#diff-abadc3d841d1da75bab0d7546cbc2e95R253?